.prettierignore

```diff
@@ -4,19 +4,8 @@ node_modules
 build
 coverage
 .docusaurus
-packages/docusaurus-utils/lib/
-packages/docusaurus-utils-validation/lib/
 packages/docusaurus/lib/
-packages/docusaurus-init/lib/
-packages/docusaurus-plugin-client-redirects/lib/
+packages/docusaurus-*/lib/
```

Comment:
I think it won't work for the v1 package?

![image](https://user-images.githubusercontent.com/749374/100921020-44ffdb00-34dc-11eb-9730-0b6d769b5074.png)

Reply:
For this is responsible `.gitignore`, which do not include v1 `lib`. I can update `.gitignore` too.

Reply:
Ah, you mean the change should NOT exclude the files from v1 `lib`, right?

Reply:
<img width="632" alt="Screenshot 2020-12-02 205904" src="https://user-images.githubusercontent.com/719641/100924722-4253b480-34e1-11eb-9125-87aa2e3ab8a9.png">

Reply:
yes :)

Reply:
https://github.com/facebook/docusaurus/pull/3865/files#diff-a0fdacd2ade87c5238dde44378f574f7e123e669acdf8b740878b14b33b20275R15

This line in ignores deals with that, so Prettier and other tools will ignore all `docusuaurs-*/lib`s but not the `docusuaurs-1.x/lib`.